crates/api_models/src/payments.rs

```diff
@@ -4700,6 +4701,15 @@ pub struct QrCodeNextStepsInstruction {
 pub struct WaitScreenInstructions {
     pub display_from_timestamp: i128,
     pub display_to_timestamp: Option<i128>,
+    pub poll_config: Option<PollConfig>,
+}
+
+#[derive(Clone, Debug, Eq, PartialEq, serde::Serialize, serde::Deserialize, ToSchema)]
+pub struct PollConfig {
+    /// Interval of the poll
+    pub delay_in_secs: i8,
+    /// Frequency of the poll
+    pub frequency: i8,
```

Comment:
Do these necessarily need to be signed integers, or can they be unsigned integers?

And by using `i8`, you're restricting these values to be a max of 127 (it would be 255 in case of `u8`). Is that intentional / sufficient?